packages/site/src/layout/NavMenu.tsx

```diff
@@ -18,116 +19,102 @@ export const NavMenu = () => {
   if (isMinimal) return null;
 
   return (
-    <div
-      style={{
-        width: 220,
-        height: "100dvh",
-        backgroundColor: "var(--color-surface--background)",
-        overflow: "auto",
-      }}
-    >
-      <Box height={24} padding="base">
-        <Link to="/">
-          <JobberLogo />
-        </Link>
-      </Box>
-      <Box padding="base">
-        <Button
-          onClick={() => setOpen(true)}
-          label="Search"
-          icon="search"
-          variation="subtle"
-        />
-      </Box>
-      <SearchBox open={open} setOpen={setOpen} />
-      <MenuList>
+    <nav className={styles.navMenuContainer}>
+      <div className={styles.navMenuHeader}>
+        <Box>
+          <Link to="/">
+            <JobberLogo />
+          </Link>
+        </Box>
         <Box>
-          {routes?.map((route, routeIndex) => {
-            if (route.inNav === false) return null;
+          <Button
+            onClick={() => setOpen(true)}
+            label="Search"
+            icon="search"
+            variation="subtle"
+          />
+        </Box>
+        <SearchBox open={open} setOpen={setOpen} />
+      </div>
+      <div className={styles.navMenu}>
+        <MenuList>
+          <Content spacing="smaller">
+            {routes?.map((route, routeIndex) => {
+              if (route.inNav === false) return null;
 
-            interface MenuItem {
-              handle: string;
-              children?: MenuItem[];
-              path?: string;
-            }
+              interface MenuItem {
```

Comment:
I could be missing something but is there a benefit to defining this interface and the below functions here as opposed to elsewhere in the file? I think that would really improve the readability over all the nesting we have at the moment

Reply:
(no thoughts or opinions about this on my end)

Reply:
I just updated this, but I was under the understanding we don't mind code that isn't completely perfect or ideal in favour of moving faster